includes/notes/class-wc-admin-notes.php

```diff
@@ -152,11 +152,17 @@ public static function unsnooze_notes() {
 	 * Schedule unsnooze notes event.
 	 */
 	public static function schedule_unsnooze_notes() {
-		$queue = WC()->queue();
-		$next  = $queue->get_next( self::UNSNOOZE_HOOK );
 
-		if ( ! $next ) {
-			$queue->schedule_recurring( time(), HOUR_IN_SECONDS, self::UNSNOOZE_HOOK, array(), self::QUEUE_GROUP );
+		$snooze_checked_transient_key = sprintf( '%_checked', self::UNSNOOZE_HOOK );
```

Comment:
```suggestion
		$snooze_checked_transient_key = sprintf( '%s_checked', self::UNSNOOZE_HOOK );
```

Reply:
This was creating a transient `checked`.

Reply:
Fixed and rebased. Thanks @rrennick! Sorry for the silly error! :)